charts/hono/values.yaml

```diff
@@ -116,48 +116,48 @@ adapters:
   # This property MUST be set if "amqpMessagingNetworkDeployExample" is set to false.
   # Please refer to https://www.eclipse.org/hono/docs/admin-guide/hono-client-configuration/
   # for a description of supported properties.
-  #amqpMessagingNetworkSpec:
-  #  host: my-amqp-host
-  #  port: 5671
-  #  trustStorePath: /etc/conf/amqp-trust-store.pem
-  #  credentialsPath: /etc/conf/amqp-credentials.properties
+  # amqpMessagingNetworkSpec:
```

Comment:
IMHO adding these whitespaces is not very helpful, because
* it makes it harder to see where the comment ends and where the property begins and
* it requires more work to uncomment the properties if needed